epoxy-litho/src/main/java/com/airbnb/epoxy/EpoxyLithoModel.java

```diff
@@ -0,0 +1,88 @@
+package com.airbnb.epoxy;
+
+import android.os.Looper;
+import android.view.ViewGroup;
+
+import com.facebook.litho.Component;
+import com.facebook.litho.ComponentContext;
+import com.facebook.litho.ComponentLifecycle;
+import com.facebook.litho.ComponentTree;
+import com.facebook.litho.LithoView;
+
+import java.util.List;
+
+/**
+ * Experiment support for Facebook's Litho library. Any litho component specs will automatically
+ * have a model generated for them that extends this class.
+ */
+public abstract class EpoxyLithoModel<T extends ComponentLifecycle>
+    extends EpoxyModelWithView<LithoView> {
+
+  private Component<T> component;
+
+  protected abstract Component<T> buildComponent(ComponentContext context);
+
+  public Component<T> getComponent(ComponentContext context) {
+    // TODO: (eli_hart 4/23/17) verify added to controller first?
+    if (component == null) {
+      // TODO: (eli_hart 4/23/17) reuse component context?
+      component = buildComponent(context);
+    }
+    return component;
+  }
+
+  public void clearComponent() {
+    // TODO: (eli_hart 4/23/17) Do this at some point to avoid leaking the context?
+    component = null;
+  }
+
+  @Override
+  public void bind(LithoView view) {
+    Component<T> component = getComponent(view.getComponentContext());
+    if (view.getComponentTree() == null) {
+      view.setComponentTree(ComponentTree.create(view.getComponentContext(), component)
+          .asyncStateUpdates(false) // does this need to be off?
```

Comment:
I'm not sure if this needs to be off since [Litho claims to allow you to access a view from any thread](https://github.com/facebook/litho/blob/master/docs/_docs/asynchronous-layout.md#sync-and-async-operations) in a safe mannor... but as I was doing some more research into the topic of recycling I came across [this line of code](https://github.com/facebook/litho/blob/master/litho-widget/src/main/java/com/facebook/litho/widget/RecyclerBinder.java#L172) which seems to show that they themselves haven't even implemented the async recycling correctly yet.